tbx/project_styleguide/templates/patterns/atoms/motif-heading/motif-heading.html

```diff
@@ -1,5 +1,24 @@
-<h{{ heading_level|default:1 }} class="motif-heading {% if classes %}{{ classes }}{% endif %}" aria-label="{% firstof aria_label heading %}">
-    {% with first_letter=heading|slice:":1" %}
-        <span class="motif-heading__drop-cap {% if first_letter|lower == 'i' %}motif-heading__drop-cap--i{% endif %} {% if first_letter|lower in 'f,p' %}motif-heading__drop-cap--narrow{% endif %} {% if first_letter|lower == 'w' %}motif-heading__drop-cap--narrower{% endif %} {% if first_letter|lower in 't,y' %}motif-heading__drop-cap--narrowest{% endif %}">{{ first_letter }}</span><span>{{ heading|slice:"1:" }}</span>
-    {% endwith %}
-</h{{ heading_level|default:1 }}>
+{% if heading_level > '1' and heading_level < '7' %}
+    {# If the heading level is anything other than one, output the heading tag with the motif styles #}
+    <h{{ heading_level|default:1 }} class="motif-heading {% if classes %}{{ classes }}{% endif %}" aria-label="{% firstof aria_label heading %}">
+{% else %}
+    {% comment %}
+    If the heading level is 1 (this is also the default which is an empty string),
+    output the h1 tag with a visually hidden style, then output a paragraph tag
+    with the motif heading styles. This is to mitigate an issue where google
+    was showing the motif headings without the first letter in search results, probably
+    because of the `color:transparent` CSS on the initial cap.
+    {% endcomment %}
+    <h{{ heading_level|default:1 }} class="sr-only">{{ heading }}</h{{ heading_level|default:1 }}>
+    <p aria-hidden="true" class="motif-heading {% if classes %}{{ classes }}{% endif %}" aria-label="{% firstof aria_label heading %}">
```

Comment:
Q: Do we still need the `aria-label` now that the element is `aria-hidden`?

Reply:
good catch